http/vibe/http/common.d

```diff
@@ -398,11 +399,17 @@ final class ChunkedInputStream : InputStream
 		readChunk();
 	}
 
-	@property bool empty() const { return m_bytesInCurrentChunk == 0; }
+	@property bool empty() { return bytesInCurrentChunk == 0; }
 
-	@property ulong leastSize() const { return m_bytesInCurrentChunk; }
+	@property ulong leastSize() { return bytesInCurrentChunk; }
 
-	@property bool dataAvailableForRead() { return m_bytesInCurrentChunk > 0 && m_in.dataAvailableForRead; }
+	@property bool dataAvailableForRead() { return bytesInCurrentChunk > 0 && m_in.dataAvailableForRead; }
```

Comment:
This must stay as `m_bytesInCurrentChunk`, so that `dataAvailableForRead` does not block (its purpose is to be able to tell whether a read will block).

Reply:
```suggestion
	@property bool dataAvailableForRead() { return m_bytesInCurrentChunk > 0 && m_in.dataAvailableForRead; }
```